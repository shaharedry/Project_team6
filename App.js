import React ,{ useState } from 'react';
import { StyleSheet, Text, View } from 'react-native';


import Navigation from './navigation/Navigation';

/*const FetchFonts =()=>{
  return Font.loadAsync({
    'open-sans': require('./assets/fonts/OpenSans-Regular.ttf'),
    'open-sans-bold': require('./assets/fonts/OpenSans-Bold.ttf')
  })
}*/


export default function App() {
  /*const [FontLoaded,setFontLoaded] = useState(false);

  if(!FontLoaded){
    return (
    <AppLoading
      startAsync={FetchFonts}
      onFinish={() => setFontLoaded(true)}
      />
      );
    } */
  return <Navigation/>;
}

const styles = StyleSheet.create({
  screen:{ 
    flex: 1
  }
});
