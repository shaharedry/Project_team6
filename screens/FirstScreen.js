import React from 'react';
import {View, Text, StyleSheet, Button} from 'react-native';
import { connect } from 'react-redux';


class FirstScreen extends React.Component {
    render(){
      return(
        <View style={styles.screen}>
            <View style={styles.buttonContainer}>
                <Button title="Sign In" onPress={() => {this.props.navigation.navigate('TypeSignin')}} color='#1e90ff'/>
            </View>
            <View style={styles.buttonContainer}>
                <Button title="Sign Up" onPress={() => {this.props.navigation.navigate('Signup')}} color='#1e90ff'/>
            </View>
        </View>
        //matandbaa
        );
      }
};

const styles = StyleSheet.create({
    screen: {
        flex: 1,
        padding : 100,
        alignItems: 'center',
        flexDirection: 'column',
        alignItems: 'center',
        justifyContent: 'center',
    },
    buttonContainer:{
        width: 250,
        height: 150,
        justifyContent: 'center',
        paddingBottom: 100 ,
        borderRadius: 10
    }
});

export default (FirstScreen)