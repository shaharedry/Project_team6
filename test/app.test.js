import React from "react";
import renderer from "react-test-renderer";
import App from '../App';
import FirstScreen from "../screens/FirstScreen";

/*test('Open first screen', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).dispatch(FirstScreen);
});

it('Test click event', () => {
    const mockCallBack = jest.fn();

    const button = shallow((<Button onClick={mockCallBack}>Ok!</Button>));
    button.find('button').simulate('click');
    expect(mockCallBack.mock.calls.length).toEqual(1);
});*/

test('renders correctly across screens', () => {
    const tree = renderer.create().toJSON();
    expect(tree).toMatchSnapshot();
}); 