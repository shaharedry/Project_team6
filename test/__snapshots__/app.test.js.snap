// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly across screens 1`] = `null`;
